bin/mlspace-cdk.ts

```diff
@@ -77,12 +77,15 @@ const websiteBucketName = `${config.WEBSITE_BUCKET_NAME}-${config.AWS_ACCOUNT}`;
 const cwlBucketName = `${config.LOGS_BUCKET_NAME}-${config.AWS_ACCOUNT}`;
 const accessLogsBucketName = `${config.ACCESS_LOGS_BUCKET_NAME}-${config.AWS_ACCOUNT}`;
 
+//Translate is not currently available in us-isob-east1
+const enableTranslate = config.AWS_REGION !== 'us-isob-east-1';
```

Comment:
We should probably make this an allowlist so that this will work for the future iso regions that might come online without Translate.

Reply:
Okay I'll swap this to a list so we can easily add/remove regions from it as needed